src/test/java/nextstep/subway/line/SectionsTest.java

```diff
@@ -0,0 +1,102 @@
+package nextstep.subway.line;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import nextstep.subway.exception.LimitDistanceException;
+import nextstep.subway.exception.CanNotAddSectionException;
+import nextstep.subway.exception.RegisteredSectionException;
+import nextstep.subway.line.domain.Section;
+import nextstep.subway.line.domain.Sections;
+import nextstep.subway.station.domain.Station;
+
+public class SectionsTest {
+    private Map<String, Station> stationMap = new HashMap<>();
+
+    @BeforeEach
+    public void setUp() {
+        List<Station> stations = new ArrayList<>();
+
+        stations.add(new Station("강남역"));
```

Comment:
```suggestion
        Station 강남역 = new Station("강남역"));
```
Map 대신에 아예 이렇게 사용해보면 어떨까요?